cg/meta/encryption/encryption.py

```diff
@@ -92,6 +130,127 @@ def get_symmetric_decryption_command(
         decryption_parameters.extend(output_parameter)
         return decryption_parameters
 
+    def create_pending_file(self, pending_path: Path) -> None:
+        """Create a pending flag file."""
+        LOG.info(f"Creating pending flag {pending_path}")
+        if self.dry_run:
+            return
+        pending_path.touch(exist_ok=False)
+
+
+class FlowCellEncryptionAPI(EncryptionAPI):
+    """Encryption functionality for flow cells."""
+
+    def __init__(
+        self,
+        config: dict,
+        binary_path: str,
+        dry_run: bool = False,
```

Comment:
```suggestion
        dry_run: bool = False,
        flow_cell: FlowCellDirectoryData
        flow_cell_encrypt_dir: Path
        ...
```
Would it be as proposed with above to move `FlowCell` specific logic from the CLI here.

Reply:
Fixed